src/libraries/System.Private.Xml/src/System/Xml/XmlConvert.cs

```diff
@@ -78,23 +79,24 @@ public static string EncodeLocalName(string name)
 
         public static string DecodeName(string name)
         {
-            if (name == null || name.Length == 0)
+            if (string.IsNullOrEmpty(name))
```

Comment:
This implies that the argument and return type should be nullable (and potentially marked with the NotNullIfNotNull attribute).